bin/entrypoint

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+./bin/setup
```

Comment:
You don't need everything inside this script (e.g "gem install bundler")

I'd just add to the Dockerfile:
RUN bin/rails db:setup
CMD ["rails", "server", "-b", "0.0.0.0"]